docs/comparisons.md

```diff
@@ -7,6 +7,10 @@ There are many tools and frameworks in the data ecosystem. This page tries to ma
 ## dbt
 [dbt](https://www.getdbt.com/) is a tool for data transformations. It is a pioneer in this space and has shown how valuable transformation frameworks can be. Although dbt is a fanstastic tool, it has trouble scaling with data and organizational size.
 
+dbt initially built their product focused on smaller companies with small data. These companies are able to fully refresh their entire warehouse every run, which usually ensures correctness. Fully refreshing your entire warehouse is not cost effective or feasible at many companies. We've designed SQLMesh to work at any data or organizational scale. This allows data teams of all sizes to compute pipelines correctly without excessives costs or wait times.
```

Comment:
This is conjecture of their intentions. Can we say something more from our perspective of what the tool was like at launch?

When dbt arrived on the scene, they brought best practices from software development to the data community, that acceled in environments where the warehouse could be fully refreshed with each run.